binding/web/src/types.ts

```diff
@@ -0,0 +1,113 @@
+/*
+  Copyright 2022 Picovoice Inc.
+
+  You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+  file accompanying this source.
+
+  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations under the License.
+*/
+
+export type RhinoInitConfig = {
+  processErrorCallback?: (error: string) => void;
+  /** @defaultValue '0.5' */
+  sensitivity?: number;
+  /** @defaultValue '1.0' */
+  endpointDurationSec?: number;
+  /** @defaultValue 'false' */
+  requireEndpoint?: boolean;
+};
+
+export type RhinoInputConfig = {
+  /** @defaultValue 'rhino_model' */
+  customWritePath?: string;
+  /** @defaultValue false */
+  forceWrite?: boolean;
+  /** @defaultValue 1 */
+  version?: number;
+};
+
+export type RhinoOptions = RhinoInitConfig & RhinoInputConfig;
+
+export type RhinoContext = {
+  /** An arbitrary label used for caching purposes */
+  label: string;
+  /** Base64 representation of a trained Rhino context (`.rhn` file) */
+  base64?: string;
+  /** The Rhino context (`.rhn` file) path relative to the public directory */
+  rhnPath?: string;
+  /** A flag that indicates whether the cached context should be used */
+  /** @defaultValue false */
+  usedCachedContext?: boolean;
```

Comment:
I don't think this variable name is accurate. It essentially enables `forceWrite` in the underlying web-utils. It should probably be called `disableCacheStorage` or something.

Reply:
If we update it here we should update it across all the bindings, since I think we use the same terminology in a few places.

Reply:
yeah I noticed. Porcupine has to get updated anyway to support the new wvp so that's fine

Reply:
Should I just make it "forceWrite" since it's the same as the model?

Reply:
sure, let's call it `forceWrite`